### DISCUSSION TITLE
Docs Overview

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/site#763] [General]
# Docs Overview

The docs module provides the basic layouts and templates for docs site, it&rsquo;s useful for writing project documentation, digital garden, users guides and tutorials.

https://hbstack.dev/modules/docs/overview/

**Comment by migliori** [comment]
The `nav_weight` works well for the main navigation items but not for nested items (e.g. inside dropdowns)
Am I wrong, or is there a workaround?
Thanks

**Reply by migliori** [reply]
... indeed it works perfectly - I had to restart the live server for some reason.

**Reply by razonyang** [reply]
> ... indeed it works perfectly - I had to restart the live server for some reason.

No problem.

**Reply by FuadEfendi** [reply]
When I click on the "Strategy" menu item under the "Behavioral" section on https://softwarepatternslexicon.com/patterns-python/behavioral/ page will show https://softwarepatternslexicon.com/patterns-python/behavioral/strategy/ and the menu on the left will reset to top (scrollbar doesn't keep "state"); so that I need to scroll down again and again; how to make "scrollbar" on the left-side menu to keep state? Thanks,

**Reply by razonyang** [reply]
That's indeed an UE issue, will check it later.

Please create a new issue when you encounter a problem in the future.

**Reply by FuadEfendi** [reply]
Ok thanks; I just didn't know on which submodule to create this issue (I think on "docs"?)

**Reply by razonyang** [reply]
This issue should be fixed since [github.com/hbstack/docs@v0.18.1](https://github.com/hbstack/docs/releases/tag/v0.18.1).

>  I just didn't know on which submodule to create this issue.

Yes, this is also an issue, since there are a lot of modules.

You can report issues on [here](https://github.com/orgs/hbstack/discussions/categories/issues-and-bugs) if you have no idea which repo/module the issue belongs to.

**Reply by FuadEfendi** [reply]
validated, it works, thank you!
